pkg/nsx/services/firewall.go

```diff
@@ -708,15 +709,15 @@ func buildPeerTags(obj *v1alpha1.SecurityPolicy, peers *[]v1alpha1.SecurityPolic
 	serializedBytes, _ := json.Marshal(*peers)
 	groupHash := util.Sha1(string(serializedBytes))
 	var peerTags = []model.Tag{
-		model.Tag{
+		{
```

Comment:
@heypnus please help confirm the change here

Reply:
@TaoZou1  what's your golang version?

Reply:
A little strange. In my env (1.17.3), it didn't report error. I'm ok to remove it because your version is fresher than mine.